backend/alert/alerts.py

```diff
@@ -48,46 +49,134 @@ def __init__(self, template, reg):
                 "auto_resubscribe": reg.auto_resubscribe,
             }
         )
+        self.close_text = None
+        if close_template:
+            t = loader.get_template(close_template)
+            self.close_text = t.render(
+                {
+                    "course": reg.section.full_code,
+                    "brand": "Penn Course Alert",
+                    "auto_resubscribe": reg.auto_resubscribe,
+                }
+            )
         self.registration = reg
 
     @abstractmethod
-    def send_alert(self):
+    def send_alert(self, close_notification=False):
         pass
 
 
 class Email(Alert):
     def __init__(self, reg):
-        super().__init__("alert/email_alert.html", reg)
+        super().__init__(
+            "alert/email_alert.html", reg, close_template="alert/email_close_alert.html"
```

Comment:
Did we decide on sending email close alerts? Not a cost issue, but thought we were keeping close alerts to mobile for now as an exclusive feature.

Reply:
So I was assuming standardization on the backend would be better and then we could regulate whether or not to afford the feature on the frontend (since close alerts are disabled by default, if the frontend doesn't change they won't be available to users on the web version).  Does this make sense or should I remove on the backend?

Reply:
Should be fine to keep it in as long as we make sure to test that case